Marlin/Configuration.h

```diff
@@ -457,6 +457,10 @@ const bool Z_MIN_PROBE_ENDSTOP_INVERTING = false; // set to true to invert the l
 
 // X and Y axis travel speed (mm/m) between probes
 #define XY_PROBE_SPEED 8000
+// Speed for the first approach when probing
+#define Z_PROBE_SPEED_1 (3.5 * 60)
```

Comment:
Perhaps `Z_PROBE_SPEED_FAST` and `Z_PROBE_SPEED_SLOW` (or `Z_PROBE_SPEED_BUMP`)?